src/main/java/org/myrobotlab/service/Polly.java

```diff
@@ -189,8 +191,15 @@ public String[] getKeyNames() {
    */
   @Override
   public AudioData generateAudioData(AudioData audioData, String toSpeak) throws IOException {
+//    if (!configured) {
+//      log.error("polly not configured yet");
+//      return null;
+//    }
+    log.error("toSpeak {}", toSpeak);

```

Comment:
error level is overkill here..  info?  or remove?

Reply:
yup